tests/test_other.py

```diff
@@ -9348,7 +9349,7 @@ def test_jsmath(self):
     for i in range(len(normal)):
       if normal[i] != jsmath[i]:
         diff += 1
-    self.assertEqual(diff, 4)
+    self.assertEqual(diff, 0 if platform.machine() == 'aarch64' else 4)
```

Comment:
I think `assertLess(diff, 5)` is probably simpler here.  WDYT @kripken ?

Reply:
I think there are two reasons to change this:

1. I'm not a fan of python's if/else ternary expressions (thats a style nit obviously).  I think this makes an already non-obvious line, even less obvious.
2. I think it makes the next needlessly fragile when I think we should be making it less fragile (what happens when another architecture or OS starts to differ here?)

Having said that, if you are too busy to fix, I agree these are both fairly minor reasons.

So.. maybe a 3 out of 10 :)